solidity/dashboard/src/components/DelegateStakeForm.jsx

```diff
@@ -113,7 +113,7 @@ const TokensAmountField = ({
   const { setFieldValue } = useFormikContext()
 
   const onAddonClick = () => {
-    setFieldValue("stakeTokens", availableToStake)
+    setFieldValue("stakeTokens", toTokenUnit(availableToStake).toString())
```

Comment:
This looks like it's still blowing up (still doesn't show an error unfortunately):
```
Warning: An unhandled error was caught during low priority validation in <Formik validate /> Error: Error: [number-to-bn] while converting number "302619.18818968797564688" to BN.js instance, error: invalid number value. Value must be an integer, hex string, BN or BigNumber instance. Note, decimals are not supported. Given value: "302619.18818968797564688"
```
Looks like BigNumber returns a decimal value, but then the string gets routed to BN, and BN only takes integers. I think maybe we need to use `toFixed(0, BigNumber.ROUND_DOWN)` instead of `toString`.